layouts/docs/docs_landing.html

```diff
@@ -3,10 +3,6 @@
 {{ $allSections  := site.Sections }} <!-- "blog" and "docs" -->
 {{ $docsSections := where $allSections "Section" "docs" }}
 
-<section class="hide-content docs-sidebar">
-  {{ partial "section-nav.html" . }}
-</section>
-
```

Comment:
on further testing I'm seeing on mobile we use the header menu to show the entire expandable nav and this change breaks that functionality.